### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Patrik-Stas**
Signed-off-by: Patrik Stas <patrik.stas@absa.africa>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,8 +10,9 @@
   - iOS, 
   - NodeJS
   - Python (looking for a maintainer)  
+- Overview of [implemented Aries protocols](./aries.md)
 - Aries Framework Rust requires [mediator agency](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0046-mediators-and-relays/README.md).
-  Currently, the only such agency is [NodeVCX Agency](https://github.com/AbsaOSS/vcxagencynode/).  
+  Currently, the only such agency is [NodeVCX Agency](https://github.com/AbsaOSS/vcxagencynode/).
```
Comment (mirgee) [inline README.md:15]: BTW, this statement seems too strong. I can easily imagine some developing another agency and being offended by this :)

File: aries.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-    * [Aries Connection Spec](https://github.com/hyperledger/aries-rfcs/tree/master/features/0160-connection-protocol)
-    * Inviter
-        * [API](./libvcx/api/connection.rs) 
-        * [State diagram](docs/states/aries/connection-inviter.puml) 
-    * Invitee
-        * [API](./libvcx/api/connection.rs) 
-        * [State diagram](docs/states/aries/connection-invitee.puml) 
+* #### Connection Protocol [`https://didcomm.org/connections/1.0/*`](https://github.com/hyperledger/aries-rfcs/tree/master/features/0160-connection-protocol)
+    * [Inviter API](./libvcx/api/connection.rs) 
+    * [Invitee API](./libvcx/api/connection.rs)
```
Comment (mirgee) [inline aries.md:3]: All these api links are invalid (should be `./libvcx/src/api/...`)

File: README.md [hunk 3]
```diff
@@ -10,8 +10,9 @@
   - iOS, 
   - NodeJS
   - Python (looking for a maintainer)  
+- Overview of [implemented Aries protocols](./aries.md)
```
Comment (mirgee) [inline README.md:13]: Awesome 👍

File: README.md [hunk 4]
```diff
@@ -10,8 +10,9 @@
...
 - Aries Framework Rust requires [mediator agency](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0046-mediators-and-relays/README.md).
-  Currently, the only such agency is [NodeVCX Agency](https://github.com/AbsaOSS/vcxagencynode/).  
+  Currently, the only such agency is [NodeVCX Agency](https://github.com/AbsaOSS/vcxagencynode/).    
   
 # Work in progress
 - This is spin off what has previously been known as LibVCX library.
```
Comment (mirgee) [inline README.md:18]: We are not a spinoff :) We are the continuation of it, an evolution :)

**Review by mirgee — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Patrik-Stas) [reply README.md:18]: Until `IndySDK libvcx` is not completely abolished, I think it's accurate to say we are spinoff / fork. I saw Evernym still recently comitting to `IndySDK libvcx`, and some users are only now migrating from `IndySDK libvcx` to `aries-vcx`

Comment (mirgee) [reply README.md:18]: OK

**Pull request merged**